Heatington/AssetManager/ProductionUnit.cs

```diff
@@ -0,0 +1,41 @@
+namespace Heatington.AssetManager;
+
+public class ProductionUnit
+{
+    public Guid Id { get; set; }
+    public string Name { get; set; }
+    public string PicturePath { get; set; }
+    private double _operationPoint;
+    public double MaxHeat { get; set; } // MW
+    public double ProductionCost { get; set; } // DKK/MWh
+    public double MaxElectricity { get; set; } // MW
+    public double GasConsumption { get; set; } // MWh(gas/oil)/MWh
+    public double Co2Emission { get; set; } // kg/MWh
+
```

Comment:
Most of the props are not required to have a setter? If so, you could increase the level of encapsulation by only define the getter?

What do you think? @KonyakB

Reply:
i agree, chenged them up